riscv-runtime/src/goldilocks.rs

```diff
@@ -20,11 +23,22 @@ impl Goldilocks {
         Self(value)
     }
 
+    /// Returns the inverse of the field element.
+    pub fn inverse(self) -> Self {
+        let mut low = self.0 as u32;
+        let mut high = (self.0 >> 32) as u32;
+        unsafe {
```

Comment:
Would it make sense to only have this function on the riscv target?

Reply:
wdym by target, the Rust rv32 target, or our riscv asm machine?

Reply:
My bad, thought this was in number